app/controllers/payment-links/metadata/metadata-form.js

```diff
@@ -76,8 +95,17 @@ class MetadataForm {
   }
 }
 
-function isNotEmpty (value) {
+function isNotEmpty (value, existingMetadata) {
   return value && value.length !== 0
 }
 
+function isValidLength (value, existingMetadata) {
```

Comment:
If these methods don't actually require the `existingMetdata` attribute it's fine to leave them out

Reply:
Done